kof-operator/webapp/collector/src/components/pages/dashboards/DashboardDetails.tsx

```diff
@@ -25,15 +25,15 @@ const DashboardDetails = <
   detailsHeaderChild
 }: DashboardData<Items, Item>): JSX.Element => {
   const { isLoading, items, selectedItem, selectItem } = store();
-
   const navigate = useNavigate();
-  const { objName } = useParams();
+  const { namespace, objName } = useParams();
+  const path: string = !namespace ? objName ?? "" : `${namespace}/${objName}`;
```

Comment:
`if a then b else c` is way simpler than `if not a then b else c`:
```suggestion
  const path: string = namespace ? `${namespace}/${objName}` : objName ?? "";
```